packages/wallets/src/components/Base/WalletButton/WalletButton.tsx

```diff
@@ -1,28 +1,33 @@
-import React, { ButtonHTMLAttributes, ReactElement } from 'react';
+import React, { ButtonHTMLAttributes, CSSProperties, ReactElement } from 'react';
 import classNames from 'classnames';
 import { TGenericSizes } from '../types';
 import { WalletText } from '../WalletText';
-import styles from './WalletButton.module.css';
+import './WalletButton.scss';
 
-interface WalletButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-    color?: 'black' | 'primary-light' | 'primary' | 'white';
+type TVariant = 'ghost' | 'outlined' | ('contained' & { color: WalletButtonProps['color'] });
```

Comment:
@aizad-deriv I don't understand how to choose variant 'contained'? What I have to pass to variant in WalletButton? Object with `color` property?

Reply:
So there is 3 different variants for the button: Contained, Outlined and Ghost. 
The most common one is Contained where the button has different background colors. However as of now the other 2 doesn't have their own stock colors.

Reply:
@aizad-deriv I'm pretty sure it's wrong type for TVariant, we have to pass there string. If you want to show with contained we have to set color we need to use another way, something like this:

```
interface IContained extends WalletButtonProps {
    variant: 'contained';
    color: CSSProperties['color'];
}

interface IOutlined extends WalletButtonProps {
    variant: 'outlined';
}

interface ComponentWalletButtonProps = IContained | IOutlined
```

Hope you understand what I mean